packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameStats.tsx

```diff
@@ -13,16 +16,34 @@ import './AdvertiserNameStats.scss';
  *
  * Use cases are to show this in My Profile and Advertiser page
  */
-const AdvertiserNameStats = () => {
-    const { data: advertiserStats } = useAdvertiserStats();
+const AdvertiserNameStats = ({ advertiserStats }: { advertiserStats: TAdvertiserStats }) => {
     const { isMobile } = useDevice();
+    const isMyProfile = getCurrentRoute() === 'my-profile';
 
-    const { blocked_by_count, daysSinceJoined, rating_average, rating_count, recommended_average } =
-        advertiserStats || {};
+    const {
+        blocked_by_count,
+        daysSinceJoined,
+        is_online,
+        last_online_time,
+        rating_average,
+        rating_count,
+        recommended_average,
+    } = advertiserStats || {};
```

Comment:
is this || {} needed? since advertiserStats is not an optional parameter, will it become undefined?

Reply:
yeah we need to have this cause advertiserStats is being passed from the parent, and its possible to be undefined